content/docs/deploy/core/upgrading.mdx

```diff
@@ -13,7 +13,13 @@ sidebar_position: 10
 
 # Upgrade guide
 
-## Since 0.22.0
+## 0.22.3
```

Comment:
It looks like we haven't included any other patch releases in the upgrade guide (which seems reasonable to me; I believe we don't intend to include any breaking changes in a patch release). Should we stick with that precedent for now?